### PR TITLE
Crt release/1.9.x

### PR DESCRIPTION
**Pull request by 99**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,250 @@
+name: build
+
+on:
+  push:
+    # Sequence of patterns matched against refs/heads
+    branches:
+    # Provide the release branch
+      - crt_release/1.9.x
```
Comment (freddygv) [inline .github/workflows/build.yml:8]: Should this be `release/1.9.x` so that it matches our release branch name?

**Review by freddygv — APPROVED** [review]
LGTM, left one non-blocking question

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+          name: metadata.json
+          path: ${{ steps.generate-metadata-file.outputs.filepath }}
+
+  build:
+    needs: get-product-version
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - {go: "1.17", goos: "linux", goarch: "386"}
```
Comment (freddygv) [inline .github/workflows/build.yml:66]: This should use go 1.15

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+      - name: Install project dependencies
+        run: |
+          go install github.com/elazarl/go-bindata-assetfs/go-bindata-assetfs@38087fe
+          go install github.com/hashicorp/go-bindata/go-bindata@bf7910a
+
+      - name: Setup with node and yarn
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```
Comment (freddygv) [inline .github/workflows/build.yml:94]: Previously this release series used node 12

**Review by freddygv — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 3 above)
Comment (99) [reply .github/workflows/build.yml:94]: updated

File: .github/workflows/build.yml (hunk 2 above)
Comment (99) [reply .github/workflows/build.yml]: updated

File: .github/workflows/build.yml (hunk 1 above)
Comment (99) [reply .github/workflows/build.yml]: Yes updated to `release/1.9.x` (kept this to trigger the current branch releases to validate that all the updates are passing)

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+
+      - name: Build
+        env:
+          GOOS: ${{ matrix.goos }}
+          GOARCH: ${{ matrix.goarch }}
+          CGO_ENABLED: 0
+          GOLDFLAGS: "${{needs.get-product-version.outputs.shared-ldflags}}"
+        run: |
+          mkdir dist out
+          go build -ldflags="$GOLDFLAGS" -o dist/ .
```
Comment (freddygv) [inline .github/workflows/build.yml:121]: Should `[ "${{ matrix.goarch }}" == "arm" ] && export GOARM=5` be added here?

Comment (99) [reply .github/workflows/build.yml:122]: thank you for catching my copypasta

**Review by freddygv — APPROVED** [review]

**Pull request merged**